src/Microsoft.ML.Data/Transforms/Normalizer.cs

```diff
@@ -238,7 +268,7 @@ public SchemaShape GetOutputSchema(SchemaShape inputSchema)
         }
     }
 
-    public sealed partial class NormalizingTransformer : OneToOneTransformerBase
+    public sealed partial class NormalizingTransformer : RowToRowTransformerBase
```

Comment:
>RowToRowTransformerBase [](start = 57, length = 23)

If only place where you need label column is training, and for applying transform you don't need it, you can just leave OneToOneTransfomerBase here.

Sorry if I create confusion, I didn't thought it through properly.

Reply:
exactly. I was able to get it to work by just making Label column active *during training*, so I was wondering  why we need RowToRowTransformerBase at all.

Will keep it as OneToOneTransformerBase

---
In reply to: [241868788](https://github.com/dotnet/machinelearning/pull/1876#discussion_r241868788) [](ancestors = 241868788)